crates/orderbook/src/api/post_order.rs

```diff
@@ -308,13 +308,16 @@ pub fn post_order(
     create_order_request().and_then(move |order: OrderCreation| {
         let orderbook = orderbook.clone();
         async move {
-            let result = orderbook.add_order(order.clone()).await;
-            match &result {
-                Ok((order_uid, quote_id)) => {
-                    tracing::debug!(%order_uid, ?quote_id, "order created")
-                }
-                Err(err) => tracing::debug!(?order, ?err, "error creating order"),
-            }
+            let result = orderbook
+                .add_order(order.clone())
+                .await
+                .map(|(order_uid, quote_metadata)| {
+                    tracing::debug!(%order_uid, ?quote_metadata, "order created");
```

Comment:
nit: logging the id and solver separately instead of debug logging the metadata struct would look nicer. (i.e. `quote_id = ..., quote_solver = ...`).